### PR TITLE
Onboard openstack-tests-list repo

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

File: core-services/prow/02_config/openstack-k8s-operators/openstack-tests-list/_prowconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    orgs:
+    - medik8s
+    - medik8s
```
Comment (droslean) [inline core-services/prow/02_config/openstack-k8s-operators/openstack-tests-list/_prowconfig.yaml:15]: This is duplicated here.

File: core-services/prow/02_config/openstack-k8s-operators/openstack-tests-list/_prowconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    orgs:
```
Comment (droslean) [inline core-services/prow/02_config/openstack-k8s-operators/openstack-tests-list/_prowconfig.yaml:13]: Try to remove the whole orgs stanza

**Comment by droslean** [comment]
Please also run `make prow-config`

File: core-services/prow/02_config/openstack-k8s-operators/openstack-tests-list/_prowconfig.yaml (hunk 2 above)
Comment (son-vyas) [reply core-services/prow/02_config/openstack-k8s-operators/openstack-tests-list/_prowconfig.yaml]: done

**Comment by son-vyas** [comment]
> Please also run `make prow-config`

done

**Pull request merged**